website/docs/language/settings/terraform-cloud.mdx

```diff
@@ -29,8 +29,7 @@ terraform {
 }
 ```
 
-If you do not specify the `hostname`, it defaults to `app.terraform.io` for Terraform Cloud. For Terraform Enterprise installations, include the [hostname](/cli/cloud/settings#hostname) configuration argument.  
 
-You cannot use the CLI integration and a [state backend](/language/settings/backends) in the same configuration; they are mutually exclusive. A configuration can only provide one `cloud` block and the `cloud` block cannot refer to named values like input variables, locals, or data source attributes.
 
-Refer to [Using Terraform Cloud](/cli/cloud) in the Terraform CLI docs for more information.
```

Comment:
Are these last three lines really good to remove? The first, in particular, seems like a nuance that often trips people up because of the confusion around TFC vs. TFE, so explicitly pointing out that the `hostname` argument is for TFE still seems relevant to me.

Reply:
Thank you for asking/pointing this out! I went back and forth on this. Here's my rationale:
- This information is available in the [actual CLI Integration docs](https://www.terraform.io/cli/cloud/settings#the-cloud-block), so it's not being deleted from the docs, it's just no longer duplicated.
- My goal with this edit is to make it very clear that this is just a pointer page. We've been having some issues with this content where folks are thinking that this IS the `cloud` block docs, and then they are pinging support for more information. I think several things were contributing to that. One of them is there's _just enough_ information for someone to maybe think this could be it. In simple use cases, it works! Anyway, the de-duplication is also about making it clearer that folks should really go to the official docs (link has been moved to be above the example).

However, I think you make a good point that folks are tripped up by TFE/TFC. And so I'm going to modify the comment in the example a bit to make it clear that this is required for TFE. I'm also going to add one sentence back in about how you can't use a `cloud` block and a backend together in the same config. That seems particularly relevant because this page is right next to/in the same section as all the backends. 

Let me know if you have any other feedback - thank you!